kolibri/plugins/management/assets/src/vue/modal/index.vue

```diff
@@ -71,22 +135,21 @@
 
   @require '~core-theme.styl'
 
-  .modal
+  .modal-overlay
     position: fixed
-    z-index: 1  // TODO: why is this necessary? fix search bar, don't add z-index
     top: 0
     left: 0
     width: 100%
     height: 100%
     background: rgba(0, 0, 0, 0.7)
-    display: table
-    transition: opacity 0.3s ease
+    // transition: opacity 0.3s ease
```

Comment:
agh that's actually supposed to be there, I was trying to see the difference with and without it.